src/code-editor/styles.scss

```diff
@@ -12,7 +12,9 @@
 
 .code-editor {
   @include styles.styles-reset;
-  display: inline-block;
+  container-type: inline-size;
+  container-name: code-editor;
```

Comment:
How well does this play with scoped CSS?

Reply:
It shouldn't be relevant. Our CSS "scoping" is just creating unique classnames with arbitrary strings. When the class is assigned to a DOM node, the container name will be applied accordingly. That being said, the container name itself is not encapsulated so there is a potential vulnerability similar to our use of internal custom properties. I suppose we could look into "scoping" our container names as well.

Reply:
FYI I updated the code to remove container name and have the containment context be implicit. No functional changes here. It just removes and concerns about name spacing.

Reply:
Now with the change I am even more confused. A deeper dive needed

1. What happens when there are multiple code editors on the page?
2. Can any 3rd party CSS cancel our implicit containment context?
3. Can we accidentally apply another context in between the root and the status bar and ruin the current setup?

Reply:
To make it more error proof, I would recommend setting container-type on the closest ancestor to avoid accidental conflicts with ace styles which we do not control

<img width="992" alt="image" src="https://github.com/cloudscape-design/components/assets/812240/2baa1e4a-a268-44d5-acd6-d94e447106e8">